samigo/samigo-app/src/webapp/jsf/author/item/itemHeadings.jsp

```diff
@@ -14,6 +14,13 @@ for (i=0; i<document.links.length; i++) {
 document.links[newindex].onclick();
 }
 
+jQuery(document).ready(function($){
+	var itemType = "${itemauthor.currentItem.itemType}";
+	if(itemType == 2 || itemType == 12) {
```

Comment:
The js formatting on the rest of this file is different to this and more in line with JSLint recommendations. A space before the argument list for  an anon funciton. A space before the opening brace. A space after for and if. It's pedantic stuff, admittedly, but makes JS easier to read, imo.